keyboards/primekb/meridian/meridian.c

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2020 Holten Campbell
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "meridian.h"
+
+//Initialize B12 for in-switch caps lock
+void keyboard_pre_init_kb(void){
+    setPinOutput(B12);
+    keyboard_pre_init_user();
+}
+
+//Initialize all RGB indicators to 'off'
+__attribute__((weak))
+void keyboard_post_init_kb(void) {
+    rgblight_setrgb_at(0, 0, 0, 0); // [..., 0] = top LED
+    rgblight_setrgb_at(0, 0, 0, 1); // [..., 1] = middle LED
+    rgblight_setrgb_at(0, 0, 0, 2); // [..., 2] = bottom LED
+}
```

Comment:
Sorry.  To clarify, you'd either want the attribute weak with the user function (so that the user can specify the function in their keymap.c, and replace this one), or you'd want to call the user function at the end here.   
Either should work just fine. 

```suggestion
__attribute__((weak))
void keyboard_post_init_user(void) {
    rgblight_setrgb_at(0, 0, 0, 0); // [..., 0] = top LED
    rgblight_setrgb_at(0, 0, 0, 1); // [..., 1] = middle LED
    rgblight_setrgb_at(0, 0, 0, 2); // [..., 2] = bottom LED
}
```
or
```suggestion
void keyboard_post_init_kb(void) {
    rgblight_setrgb_at(0, 0, 0, 0); // [..., 0] = top LED
    rgblight_setrgb_at(0, 0, 0, 1); // [..., 1] = middle LED
    rgblight_setrgb_at(0, 0, 0, 2); // [..., 2] = bottom LED
    keyboard_post_init_user();
}
```

Reply:
ah I guess I overlooked it in the last suggestion. Done.